gms/gossiper.cc

```diff
@@ -2129,6 +2130,10 @@ future<> gossiper::do_shadow_round(std::unordered_set<gms::inet_address> nodes,
         std::unordered_set<gms::inet_address> nodes_talked;
         auto start_time = clk::now();
         std::list<gms::gossip_get_endpoint_states_response> responses;
+
+        utils::get_local_injector().inject("stop_during_gossip_shadow_round",
```

Comment:
note that here the shadow round hasn't even really started yet, we didn't send any messages or update any state. So this is basically "stop **before** shadow round".